src/main.rs

```diff
@@ -17,7 +17,7 @@ use caps::{Capability, CapSet};
 use caps::securebits::set_keepcaps;
 
 use serde::Serialize;
-use serde_json::{self,json};
+use serde_json::{self};
```

Comment:
Just `serde_json` without the `self` is fine.

Reply:
Changed in 5dd2c6dced2ee0f05da95803835c2f2a040938c7

Reply:
How about
``` rs
self.directory.clone().unwrap_or_else(||Path::new(".").into()).display(),
```